ui/apps/platform/src/Containers/AccessControl/AuthProviders/AuthProviderForm.tsx

```diff
@@ -237,7 +237,7 @@ function AuthProviderForm({
     const { dirty, handleChange, isValid, setFieldValue, handleBlur, values, errors, touched } =
         formik;
 
-    function onChange(_value, event) {
+    function onChange(event: React.FormEvent) {
```

Comment:
The params here were inferred any, leading to an error. We only need the event so the other param can be removed and `onChange` can be passed directly to the component. There are a handful of other identical changes further down.